tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -366,27 +379,39 @@ def _ValidateEntityBlock(self, block: syaml.YAML) -> None:
 
     for key in block.keys():
       if key in self._validated_entities:
-        raise ValueError('Duplicate key {0}'.format(key))
+        raise ValueError("Duplicate key '{0}'".format(key))
       self._ValidateEntityContent(block.get(key))
     self._validated_entities.update(block.data)
 
-  def _ValidateBlock(self, unvalidated_block: str,
+  def _ValidateBlock(self, unvalidated_block: {},
```

Comment:
use specific type when created - note, I think the python version here might still require the typename to be 'quoted', but try without first